app/controllers/spree/admin/base_controller.rb

```diff
@@ -81,6 +84,12 @@ def check_json_authenticity
 
       private
 
+      def page_load_request?
+        return false if request.format.include?('turbo')
+
+        html_request?
+      end
+
       def html_request?
```

Comment:
Keeping this helper because there might be some cases where we would want to check for requests that display data (HTML, turbo, etc) rather than serve data (JSON, XML, etc).
Moreover, it's being used here:

https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/controllers/admin/customers_controller.rb#L8